iXBRLViewerPlugin/viewer/src/js/reportset.js

```diff
@@ -101,12 +101,12 @@ export class ReportSet {
 
     namespaceGroups() {
         const counts = {};
-        $.each(this.facts(), function (i, f) {
+        this.facts().forEach((f, i) => {
```

Comment:
Out of curiosity, why `forEach` here rather than `for (const f of this.facts())`?

Pre-existing, I guess `counts[f.conceptQName().prefix] = 1 + (counts[f.conceptQName().prefix] || 0)` saves a prop lookup.

Reply:
> Out of curiosity, why `forEach` here rather than `for (const f of this.facts())`?

I'm not really sure why I did that.  Have switched in 2d4aec6 and tidied up the increment too.